CesiumGltf/generated/include/CesiumGltf/AccessorSparse.h

```diff
@@ -36,5 +36,13 @@ struct CESIUMGLTF_API AccessorSparse final
    * accessor values.
    */
   CesiumGltf::AccessorSparseValues values;
+
+  int64_t getSizeBytes() const {
+    int64_t accum = 0;
+    accum += sizeof(this->count);
+    accum += this->indices.getSizeBytes();
+    accum += this->values.getSizeBytes();
+    return accum;
```

Comment:
The size should include the size of the ExtensibleObject base class as well. Extensions and extras could be significant in some cases.

Also, counting the size of each individual field like this won't account for padding added into the structure. It's better to get the size of the entire structure, for example, `sizeof(AccessorSparse)`. Then, as you're doing, you need to add in the size of any additional data that not stored directly in a field, and that's where it gets tricky, because you need to avoid double counting. I think the most accurate implementation looks something like this:

```suggestion
    int64_t accum = 0;
    accum += sizeof(AccessorSparse);
    accum += ExtensibleObject::getSizeBytes() - sizeof(ExtensibleObject);
    accum += this->indices.getSizeBytes() - sizeof(AccessorSparseIndices);
    accum += this->values.getSizeBytes() - sizeof(AccessorSparseValues);
    return accum;
```

Basically, when you're counting the size of `indices`, you need to avoid double-counting the `AccessorSparseIndices` structure itself, because it's included in both `sizeof(AccessorSparse)` and in `this->indices.getSizeBytes()`.